pkg/authorization/registry/subjectaccessreview/rest.go

```diff
@@ -99,15 +99,14 @@ func (r *REST) Create(ctx kapi.Context, obj runtime.Object) (runtime.Object, err
 	requestContext := kapi.WithNamespace(kapi.WithUser(ctx, userToCheck), subjectAccessReview.Action.Namespace)
 	attributes := authorizer.ToDefaultAuthorizationAttributes(subjectAccessReview.Action)
 	allowed, reason, err := r.authorizer.Authorize(requestContext, attributes)
-	if err != nil {
```

Comment:
> I guess if there's an internal error, I'd expect an internal error, not a SubjectAccessReviewResponse, but maybe it's fine

I don't think you would.  Above this call, sure.  At this point, I'd expect to get the `authorize` result